src/ui/pages/SwapForm/Quotes/styles.module.css

```diff
@@ -0,0 +1,12 @@
+.gradientText {
```

Comment:
Should we extract this style to helperStyles.css?

Reply:
currently we have like 3 differnt gradientText CSS classes, but they are all a bit different

Reply:
This is true :((
Let's wait for the design update for this one anyway

Reply:
I think it's absolutely fine to duplicate these